app/models/user.rb

```diff
@@ -15,6 +17,9 @@ class User < ApplicationRecord
                     length: {minimum: Settings.validations.length.password_min},
                     allow_nil: true
 
+  scope :ordered_by_name, ->{order(name: :asc)}
```

Comment:
```suggestion
  scope :ordered_by_name, ->{order :name}
```
asc là mặc định rồi em nhé